### ISSUE TITLE
Question regarding fine-tuning learning rate

### ISSUE THREAD

**Issue by axyong2** [microsoft/mattergen#53]
Hi, 

The paper states that the learning rate used for fine-tuning was 6e-5, but the fine-tuning learning rate provided in this repo is 5e-6 ([https://github.com/microsoft/mattergen/blob/74b9fcab2b5140b4e41001ba800c1c2170bafe8d/mattergen/conf/finetune.yaml](url)). I would like to ask if the learning rate used for the paper was 6e-5 or 5e-6? Thank you for taking the time to answer my question!

**Comment by danielzuegner** [comment]
Hi @axyong2,

thanks for raising this. I've confirmed that for the paper we were using 6e-5, while for the re-training for this repo we (by mistake) used 5e-6. We will re-train the models 6e-5 and update the checkpoints in this repo.

Best,

Daniel
